app/styles/partials/footer.scss

```diff
@@ -33,3 +33,68 @@ footer {
     right: 0 !important;
   }
 }
+
+
+@media screen and (min-width : 1200px) {
+    #main-container {
```

Comment:
@niranjan94 What are those pages with a side rails? I am not able to find any other pages for now who are having rails.